doc/neo-tree.txt

```diff
@@ -863,6 +863,7 @@ NeoTreeSignColumn         |hl-SignColumn| override in Neo-tree window.
 NeoTreeStatusLine         |hl-StatusLine| override in Neo-tree window.
 NeoTreeStatusLineNC       |hl-StatusLineNC| override in Neo-tree window.
 NeoTreeVertSplit          |hl-VertSplit| override in Neo-tree window.
+NeoTreeWinSeparator       |hl-WinSeparator override in Neo-tree window.
```

Comment:
This is missing a closing `|` at the end of the link.

Reply:
🤦